gym/wrappers/normalize_action.py

```diff
@@ -0,0 +1,19 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class NormalizeAction(gym.ActionWrapper):
```

Comment:
Should this be named `RescaleAction`? Normalization is a bit confusing because the rescaling is not data dependent.

Reply:
@hartikainen @pzhokhov I like this idea ! That would allow more general range of the actions that user can produce prior to passing to the environment.

Reply:
@hartikainen @pzhokhov Done